src/Nest/Search/Suggesters/Suggest/SuggestRequest.cs

```diff
@@ -1,31 +1,35 @@
 ﻿using System;
-using System.Collections.Generic;
 using Newtonsoft.Json;
 
 namespace Nest
 {
+	[Obsolete("Removed in NEST 6.x")]
 	[JsonConverter(typeof(SuggestRequestJsonConverter))]
 	public partial interface ISuggestRequest
 	{
 		string GlobalText { get; set; }
 		ISuggestContainer Suggest { get; set; }
 	}
 
+	[Obsolete("Removed in NEST 6.x")]
 	public partial class SuggestRequest
 	{
 		public string GlobalText { get; set; }
 		public ISuggestContainer Suggest { get; set; }
 	}
 
+	[Obsolete("Removed in NEST 6.x")]
 	public partial interface ISuggestRequest<T> : ISearchRequest { }
 
+	[Obsolete("Removed in NEST 6.x")]
```

Comment:
I think we can help users here by suggesting that they move to using suggesters on the `Search()` endpoint.

Reply:
:+1: